pkg/upgrade/v1_22_0.go

```diff
@@ -30,16 +31,45 @@ func upgrade1_22_0(ctx context.Context, client client.Client, jaeger v1.Jaeger)
 		},
 	}
 
-	flagsAll := []deprecationFlagMap{{
-		from: "cassandra.tls.verify-host",
-		to:   "cassandra.tls.skip-host-verify",
-	}}
-
 	j := &jaeger
 	j.Spec.AllInOne.Options = migrateDeprecatedOptions(j, j.Spec.AllInOne.Options, flagMapCollector)
 	j.Spec.Collector.Options = migrateDeprecatedOptions(j, j.Spec.Collector.Options, flagMapCollector)
 	j.Spec.Agent.Options = migrateDeprecatedOptions(j, j.Spec.Agent.Options, flagMapAgent)
 	j.Spec.Query.Options = migrateDeprecatedOptions(j, j.Spec.Query.Options, flagMapQuery)
 
-	return migrateAllDeprecatedOptions(jaeger, flagsAll), nil
+	return migrateCassandraVerifyFlag(jaeger), nil
+}
+
+func flagBoolValue(v interface{}) bool {
```

Comment:
You should either move this to a generic location or prefix it with the version number.